web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
Enter the following into one of your `description` fields, then go to one of the `show` or `index` pages:

```
<script>
  alert("Haxed");
</script>
```